server/src/routes/users.routes.ts

```diff
@@ -60,6 +64,50 @@ usersRouter.post('/', async (request, response, next) => {
 
 usersRouter.use(ensureAuthenticated); // All user editing routes (below) require authentication
 
+/**
+ * Endpoint for export data from a given User.
+ */
+ usersRouter.get('/export', async (request, response, next) => {
+  try {
+    const user_id = request.user.id;
+
+    const userRepository = getRepository(User)
+    const findUser: User | undefined = await userRepository.findOne(user_id);
+
+    if (findUser) {
+      // Find all notifications related to findUser
+      const notificationsRepository = await getRepository(Notification);
+      const notifications = await notificationsRepository.find({
+        where: { user: findUser },
+        relations: ['user'],
+      });
+
+      // Get relevant data from User object + notifications
+      const exportedUserData = {
+        id: findUser.id,
+        email: findUser.email,
+        notify: findUser.notify,
+        notifyByEmail: findUser.notifyByEmail,
+        notifyByBrowser: findUser.notifyByBrowser,
+        notifications: notifications,
```

Comment:
How to I test for created notifications? I haven't ever created those in the server.

Reply:
Yep, looks good (to me :pleading_face:) will test that up later